pulumi/bin/subset_of_artifacts.jq

```diff
@@ -0,0 +1,8 @@
+. 
+| with_entries(
+    select(
+        # Select keys that match this regex.
+        # (note: match also supports N things to match on, makes adding to this subset easy)
+        .key|match("grapl-nomad-consul-*")
```

Comment:
I'm not quite sure of the utility of this; why not just pull the whole `artifacts` object over?